sdk/CMakeLists.txt

```
@@ -8,7 +8,9 @@ project("McmDPSdk" C)
 set(VERSION "0.1.0")
```

Also probably if we make changes to the SDK the version should be upgraded.

```suggestion
set(VERSION "0.1.1")
```

---

Version bumped in 37b407a07eed017bf25a323b259f1dccbb21649f